### DISCUSSION TITLE
Incorrect Profile Redirect

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#15033] [Q&A]
![osu_2021-10-10_17-16-17](https://user-images.githubusercontent.com/54728307/136713271-42d2122c-5cd9-452e-a742-b4d1f272bd3e.jpg)

Greetings, I came across a simple error, when clicking the profile in which a player mapped, it will redirect to your own profile rather than the mappers. Simple fix, but yeah.

**Comment by peppy** [comment]
Same issue as https://github.com/ppy/osu/issues/5966.

**Reply by 19peytonsmith** [reply]
This is not related to local scores. No matter what map it is, when you click on the mapped by [MAPPER] link, it will redirect to the player's profile rather than the mapper's profile.

**Reply by peppy** [reply]
Unless this is a regression, the underlying issue is the same as the linked issue, which is how we are storing/restoring user data. If so, it will be resolved with some later work (after realm is integrated fully).

**Reply by peppy** [reply]
Can confirm this is not a regression. Works fine here, which means your imported beatmaps are just in a bad state. This will eventually be fixed when we get some kind of consistency pass in.
